Sources/MapboxMaps/Ornaments/OrnamentsManager.swift

```diff
@@ -62,13 +67,14 @@ public class OrnamentsManager: NSObject {
         view.addSubview(compassView)
 
         // Info Button
-        attributionButton = MapboxInfoButtonOrnament()
-        attributionButton.attributionDataSource = attributionDataSource
+        attributionButton = InfoButtonOrnament()
         attributionButton.translatesAutoresizingMaskIntoConstraints = false
         view.addSubview(attributionButton)
 
         super.init()
 
+        attributionButton.delegate = self
```

Comment:
If we made the dialog manager the delegate, would that simplify things at all?

Reply:
Yep, this was a left over from an earlier iteration. Though it might be useful for testing purposes.